### PR TITLE
Footer/links

### PR DESCRIPTION
**Pull request by tammalee**
## Notes

I have no idea how we're supposed to do cross-browser testing on this site when it only allows a preview in Chrome. Any ideas?

### CODE REVIEW

File: styles/_footer.scss [hunk 1]
```diff
@@ -1,20 +1,195 @@
...
-  padding: 30px 0;
+footer {
+  margin: 0 auto;
+  @include fluid-value( padding-top, $min-width, $max-width, $whitespace-medium, $whitespace-medium--desktop );
+  max-width: $max-width-page-container;
+  background-color: $primary--light;
+  color: $white;
 
-  a { color: $secondary-text-color; }
+  .footer-inner {
```
Comment (eddysims) [inline styles/_footer.scss]: Is there a reason you didn't use regular `BEM` here?

File: styles/_footer.scss [hunk 2]
```diff
@@ -1,20 +1,195 @@
...
+    display: block;
 
-  &-inner {
-    @include max-width-container;
-    display: flex;
-    justify-content: space-between;
+    &-column-wrap {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 3]
```diff
@@ -1,20 +1,195 @@
...
+      @include fluid-value( margin-top, $min-width, $max-width, $whitespace-small, $whitespace-small--desktop );
+      
+      @include tablet{
```
Comment (eddysims) [inline styles/_footer.scss]: does the linter not catch this missing space?

File: styles/_footer.scss [hunk 4]
```diff
@@ -1,20 +1,195 @@
...
+        justify-content: space-between;
+      }
+    }
   }
 
-  &-language-selector {
-    color: $secondary-text-color;
-    display: inline-block;
-    font-weight: $font-weight-light;
+  .footer-logo-link {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 5]
```diff
@@ -1,20 +1,195 @@
...
+    box-shadow: unset;
+    
+    &:hover,
+    &:focus,
+    &:active {
+      box-shadow: unset;
+    }
+  }
+
+  .footer-logo {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 6]
```diff
@@ -1,20 +1,195 @@
 /***** Footer *****/
-.footer {
-  border-top: 1px solid $border-color;
-  margin-top: 60px;
-  padding: 30px 0;
+footer {
```
Comment (eddysims) [inline styles/_footer.scss]: Can we target a class here? `footer` is pretty generic

File: styles/_footer.scss [hunk 7]
```diff
@@ -1,20 +1,195 @@
...
+      box-shadow: unset;
+    }
+  }
+
+  .footer-logo {
+    color: $white;
+    max-width: 242px;
+  }
+
+  .footer {
```
Comment (eddysims) [inline styles/_footer.scss]: I see what you did here, probably not the best to havea `.footer` when at the point of this pr there was no `.footer` class. That could come off as super confusing

File: styles/_footer.scss [hunk 8]
```diff
@@ -1,20 +1,195 @@
...
+
+    &-title {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 9]
```diff
@@ -1,20 +1,195 @@
...
+      text-transform: uppercase;
+      margin-top: $spacing--desktop;
+      margin-bottom: 10px;
+
+      @include tablet {
+        margin-top: 0;
+      }
+    }
+
+    &-list {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 10]
```diff
@@ -1,20 +1,195 @@
...
+    
+        &:hover,
+        &:active,
+        &:focus {
+          box-shadow: 0 -12px 0 0 rgba($secondary, 0.25) inset;
+        }
+      }
+    }
+
+    &-social {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 11]
```diff
@@ -1,20 +1,195 @@
...
+
+      &-list {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 12]
```diff
@@ -1,20 +1,195 @@
...
+        margin: $spacing 0 0 0;
+        padding: 0;
+        list-style: none;
+        display: flex;
+        align-items: center;
+        
+        @include tablet{
```
Comment (eddysims) [inline styles/_footer.scss]: again missing space?

File: styles/_footer.scss [hunk 13]
```diff
@@ -1,20 +1,195 @@
...
+      &__item {
+        padding-right: $spacing;
+
+        @include tablet {
+          padding-right: 0;
+        }
+      }
+      
+      &__icon {
+        width: 32px;
```
Comment (eddysims) [inline styles/_footer.scss]: this width seems like it should be common?

File: styles/_footer.scss [hunk 14]
```diff
@@ -1,20 +1,195 @@
...
+          g,
+          path,
+          svg {
+            fill: $white;
+          }
+        }
+      }
+    }
+  }
+  .app-badges,
```
Comment (eddysims) [inline styles/_footer.scss]: missing space

File: styles/_footer.scss [hunk 15]
```diff
@@ -1,20 +1,195 @@
...
+      display:block;
+    
+      &:hover,
+      &:focus,
+      &:active {
+        box-shadow: unset;
+      }
+    }
+  }
+  .app-badges {
```
Comment (eddysims) [inline styles/_footer.scss]: missing space

File: styles/_footer.scss [hunk 16]
```diff
@@ -1,20 +1,195 @@
...
+      margin-top: 0;
+    }
+
+    &__badge {
+      width: 170px;
+      min-width: 170px;
+    }
+  }
+
+  .footer-copyright {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 17]
```diff
@@ -1,20 +1,195 @@
...
+    @include fluid-value( margin-top, $min-width, $max-width, $whitespace-small, $whitespace-small--desktop );
+    border-top: solid $border--width $white;
+    @include fluid-value( padding-top, $min-width, $max-width, $spacing, $whitespace-small );
+    @include fluid-value( padding-bottom, $min-width, $max-width, $spacing, $whitespace-small );
+    
+    @include tablet {
+      line-height: 1;
+    }
+
+    &-list {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 18]
```diff
@@ -1,20 +1,195 @@
...
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      @include tablet {
+        text-align: right;
+      }
+    }
+
+    &-list-item {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: styles/_footer.scss [hunk 19]
```diff
@@ -1,20 +1,195 @@
...
+
+        &:not(:last-child) {
+          padding-right: $spacing;
+          border-right: solid $border--width $white;
+          margin-right: $spacing;  
+        }
+      }
+    }
+
+    &-list-link {
```
Comment (eddysims) [inline styles/_footer.scss]: `BEM`ing

File: templates/footer.hbs [hunk 20]
```diff
@@ -1,22 +1,175 @@
...
...c0-.504.407-.913.909-.913h1.765a.91.91 0 01.815.506l4.115 8.305c.154.31.469.506.814.506h7.305a.914.914 0 00.8-1.348M36.97 25.379c0 6.5-5.244 11.77-11.713 11.77H8.65a.382.382 0 01-.381-.383v-22.01c0-3.56 2.872-6.448 6.416-6.448h11.141c.266 0 .45.267.358.517l-.648 1.75a.382.382 0 01-.357.25h-9.954c-2.46 0-4.453 2.003-4.453 4.474V34.25c0 .212.171.384.382.384h14.103c5.086 0 9.21-4.144 9.21-9.255V5.896c0-.342.41-.513.651-.271l1.74 1.749c.072.072.113.17.113.27V25.38zm-23.404.74V15.3c0-.92.743-1.667 1.66-1.667h10.768c.21 0 .381.172.381.383v10.821c0 .92-.742 1.667-1.659 1.667H13.948a.383.383 0 01-.382-.383zM5.475 14.757v19.483a.381.381 0 01-.652.271l-1.74-1.749a.385.385 0 01-.112-.271V14.756c0-6.5 5.244-11.77 11.714-11.77H31.29c.211 0 .382.17.382.382v22.01c0 3.562-2.873 6.448-6.416 6.448H14.115a.383.383 0 01-.357-.517l.647-1.749a.382.382 0 01.358-.25h9.953c2.46 0 4.453-2.003 4.453-4.474V5.884a.382.382 0 00-.381-.383H14.685c-5.087 0-9.21 4.143-9.21 9.255zM33.984.515a1.142 1.142 0 00-.81-.337h-18.49C6.673.178.178 6.705.178 14.756v18.579c0 .305.12.597.335.813l5.446 5.472c.214.216.506.337.81.337h18.489c8.012 0 14.508-6.527 14.508-14.578V6.799c0-.304-.121-.597-.336-.812L33.984.515z"/></g></svg>
```
Comment (eddysims) [inline templates/footer.hbs]: Can you include this SVG in another way? can we have like a `_logo.hbs` file that gets included? That would make the logo itself more reusable.

**Comment by eddysims** [comment]
Footer.hbs has a totally different naming standard then anything we have done before. Is this for a reason?

**Comment by eddysims** [comment]
825px wide is hard to look at.

image

File: templates/footer.hbs (hunk 20 above)
Comment (tammalee) [reply templates/footer.hbs]: Nope. I did try it but it looks like SVG still has the limitation that is needs to be inline if you want to style it.

File: styles/_footer.scss (hunk 1 above)
Comment (tammalee) [reply styles/_footer.scss]: I was adapting their code. In retrospect, if I'm going to touch a file I really should convert it to our style.

File: styles/_footer.scss (hunk 3 above)
Comment (tammalee) [reply styles/_footer.scss]: It did not! Help?

**Comment by tammalee** [comment]
> 825px wide is hard to look at.
> 
> image

Try now?
Screenshot 2019-12-17 08 41 32

File: styles/_variables.scss [hunk 21]
```diff
@@ -93,3 +69,45 @@ $header__height--min: 80px;
 $header__height--max: 138px;
 $header__padding--max: 16px;
 $header__icon-menu--dimension: 40px;
+
+/* ZENDESK DEFAULTS */
+// Colors
+$brand_color: #0072ef !default;
+$brand_text_color: #fff !default; //Color for text with an accent color background
+$text_color: #333 !default;
+$link_color: #0072ef !default;
```
Comment (eddysims) [inline styles/_variables.scss]: not a blocker, but is there a reason you use `-` for some variables and `_` for others?

File: templates/footer.hbs [hunk 22]
```diff
@@ -1,22 +1,132 @@
...
+            <a class="footer-list__link" href="tel:+18887211115">1-888-721-1115</a>
+          </li>
+          <li class="footer-list__item">
+            <a class="footer-list__link" href="mailto:hello@getjobber.com">hello@getjobber.com</a>
+          </li>
+          <li class="footer-list__item">
+            <ul class="footer-social__list">
+              <li class="footer-social__item">
+                <a class="footer-social__link" href="https://www.facebook.com/GetJobber" target="_blank" rel="noopener noreferrer">
+                  <svg class="footer-social__icon footer-social__icon--facebook" viewBox="0 0 32 32" title="Jobber Facebook" xmlns="http://www.w3.org/2000/svg"><path d="M1.334 0C.597 0 0 .597 0 1.333v29.334C0 31.403.597 32 1.334 32H17V20h-4s-.889-3.889 0-5c.889-1.111 4 0 4 0v-4c0-4.133 2.617-6.167 6.304-6.167 1.766 0 3.284.132 3.726.19v4.32l-2.557.001c-2.005 0-2.473.953-2.473 2.351V15h5l-1 5h-4l.08 12h8.587c.736 0 1.333-.597 1.333-1.333V1.333C32 .597 31.403 0 30.667 0H1.334z" fill="#99A9B0" fill-rule="evenodd"/></svg>
```
Comment (eddysims) [inline templates/footer.hbs]: You have created the svg icons, but didnt use them here?

File: styles/_footer.scss [hunk 23]
```diff
@@ -1,20 +1,195 @@
...
+  margin: 0 auto;
+  @include fluid-value( padding-top, $min-width, $max-width, $whitespace-medium, $whitespace-medium--desktop );
+  max-width: $max-width-page-container;
+  background-color: $primary--light;
+  color: $white;
 
-  a { color: $secondary-text-color; }
+  &__inner {
+    display: block;
+  }
```
Comment (eddysims) [inline styles/_footer.scss]: do you need this one? Adding a display block to a div

File: templates/footer.hbs [hunk 24]
```diff
@@ -1,22 +1,132 @@
...
...c0-.504.407-.913.909-.913h1.765a.91.91 0 01.815.506l4.115 8.305c.154.31.469.506.814.506h7.305a.914.914 0 00.8-1.348M36.97 25.379c0 6.5-5.244 11.77-11.713 11.77H8.65a.382.382 0 01-.381-.383v-22.01c0-3.56 2.872-6.448 6.416-6.448h11.141c.266 0 .45.267.358.517l-.648 1.75a.382.382 0 01-.357.25h-9.954c-2.46 0-4.453 2.003-4.453 4.474V34.25c0 .212.171.384.382.384h14.103c5.086 0 9.21-4.144 9.21-9.255V5.896c0-.342.41-.513.651-.271l1.74 1.749c.072.072.113.17.113.27V25.38zm-23.404.74V15.3c0-.92.743-1.667 1.66-1.667h10.768c.21 0 .381.172.381.383v10.821c0 .92-.742 1.667-1.659 1.667H13.948a.383.383 0 01-.382-.383zM5.475 14.757v19.483a.381.381 0 01-.652.271l-1.74-1.749a.385.385 0 01-.112-.271V14.756c0-6.5 5.244-11.77 11.714-11.77H31.29c.211 0 .382.17.382.382v22.01c0 3.562-2.873 6.448-6.416 6.448H14.115a.383.383 0 01-.357-.517l.647-1.749a.382.382 0 01.358-.25h9.953c2.46 0 4.453-2.003 4.453-4.474V5.884a.382.382 0 00-.381-.383H14.685c-5.087 0-9.21 4.143-9.21 9.255zM33.984.515a1.142 1.142 0 00-.81-.337h-18.49C6.673.178.178 6.705.178 14.756v18.579c0 .305.12.597.335.813l5.446 5.472c.214.216.506.337.81.337h18.489c8.012 0 14.508-6.527 14.508-14.578V6.799c0-.304-.121-.597-.336-.812L33.984.515z"/></g></svg>
```
Comment (eddysims) [inline templates/footer.hbs]: I saw you pulled the other svgs into assets, can we do the same here?

File: templates/footer.hbs [hunk 25]
```diff
@@ -1,22 +1,132 @@
...
+                </a>
+                </li>
+              <li class="footer-social__item">
+                <a class="footer-social__link" href="https://www.youtube.com/user/GetJobber" target="_blank" rel="noopener noreferrer">
+                  <svg class="footer-social__icon footer-social__icon--youtube" viewBox="0 0 32 23" title="Jobber Youtube" xmlns="http://www.w3.org/2000/svg"><path d="M12.696 15.404l-.001-8.991 8.646 4.511-8.645 4.48zM31.68 4.855s-.313-2.205-1.272-3.176C29.191.404 27.827.398 27.202.324 22.724 0 16.007 0 16.007 0h-.014S9.276 0 4.798.324c-.626.074-1.989.08-3.207 1.355C.632 2.65.32 4.855.32 4.855S0 7.445 0 10.034v2.428c0 2.59.32 5.179.32 5.179s.312 2.205 1.271 3.176c1.218 1.275 2.817 1.235 3.529 1.368 2.56.246 10.88.322 10.88.322s6.724-.01 11.202-.334c.625-.075 1.989-.081 3.206-1.356.959-.971 1.272-3.176 1.272-3.176s.32-2.589.32-5.179v-2.428c0-2.589-.32-5.179-.32-5.179z" fill="#99A9B0" fill-rule="evenodd"/></svg>
+                </a>
+                </li>
```
Comment (eddysims) [inline templates/footer.hbs]: some tabbing weirdness here with the `</li>`

**Comment by eddysims** [comment]
We really want people to download these apps haha.

image

**Comment by eddysims** [comment]
Im assuming that Kas has approved the no app icons on desktop

File: templates/footer.hbs (hunk 22 above)
Comment (tammalee) [reply templates/footer.hbs]: Screenshot 2019-12-17 11 13 44

Just doesn't work, brah

**Comment by tammalee** [comment]
> We really want people to download these apps haha.
> 
> image

Screenshot 2019-12-17 13 26 07

File: styles/_footer.scss (hunk 23 above)
Comment (tammalee) [reply styles/_footer.scss]: Removed!

**Pull request merged**